dirsrvtests/tests/suites/memberof_plugin/fixup_test.py

```diff
@@ -10,17 +10,146 @@
+
+logging.getLogger(__name__).setLevel(logging.DEBUG)
 log = logging.getLogger(__name__)
 
+group_name = 'group'
+member_name = 'member'
+
+@pytest.fixture(scope="function")
+def prepare_entries(topology_st, request):
+    groups = Groups(topology_st.standalone, DEFAULT_SUFFIX)
+    users = nsUserAccounts(topology_st.standalone, DEFAULT_SUFFIX)
+
+    log.info('Create group cn=group,ou=groups,{}'.format(DEFAULT_SUFFIX))
+    if groups.exists(group_name):
+        test_group = groups.get(group_name)
+        test_group.delete()
+
+    properties = {
+        'cn': group_name,
+        'description': group_name
+    }
+    test_group = groups.create(properties=properties)
+
+    log.info('Create user cn=member,ou=People,{}'.format(DEFAULT_SUFFIX))
+    if users.exists(member_name):
+        member = users.get(member_name)
+        member.delete()
+
+    properties_member = {
+        'uid': member_name,
+        'cn': member_name,
+        'displayName': member_name,
+        'uidNumber': '1000',
+        'gidNumber': '1000',
+        'homeDirectory': '/home/{}'.format(member_name)
+    }
+    member = users.create(properties=properties_member)
+
+    def fin():
+        log.info('Delete test group')
+        if test_group.exists():
+            test_group.delete()
+        log.info('Delete member user')
+        if member.exists():
+            member.delete()
+
+    request.addfinalizer(fin)
+
+
+def test_memberof_fixup_task(topology_st, prepare_entries):
+    """ Test memberOf fixup task
+
+    :id: 7e5ef63a-36c1-4cb9-9f0a-f92e4d1937ca
+    :setup: Standalone instance
+    :steps:
+        1. Create group entry: cn=group,ou=groups,dc=example,dc=com
+        2. Create member user entry: uid=member,ou=people,dc=example,dc=com
+        3. Update member user entry to have objectclass nsMemberOf
+        4. Update member user entry to contain attribute memberOf of the group entry
+        5. Enable MemberOf Plugin
+        6. Verify the member user entry still has memberOf attribute set
+        7. Verify the group entry does not have member attribute set
+        8. Run the fixupmemberof task
+        9. Verify the member user no longer contains memberOf attribute
+    :expectedresults:
+        1. Success
+        2. Success
+        3. Success
+        4. Success
+        5. Success
+        6. Success
+        7. Success
+        8. Success
+        9. Success
+    """
+
+    def _enable_disable_mbo(value):
+        """
+            Enable or disable mbo plugin depending on 'value' ('on'/'off')
+        """
+        # enable/disable the mbo plugin
+        if value == 'on':
+            topology_st.standalone.plugins.enable(name=PLUGIN_MEMBER_OF)
+        else:
+            topology_st.standalone.plugins.disable(name=PLUGIN_MEMBER_OF)
+
+        log.info("MemberOf plugin: {}".format(value))
+
+        topology_st.standalone.stop(timeout=120)
+        time.sleep(1)
+        topology_st.standalone.start(timeout=120)
+        time.sleep(3)
```

Comment:
I'm not sure I understand the purpose of this code snippet... Why not just a restart?